common/txmgr/fee/models.go

```diff
@@ -0,0 +1,22 @@
+package fee // renamed Fee from fee for chain agnostic purposes
+
+import (
+	"math/big"
+
+	"github.com/shopspring/decimal"
+	bigmath "github.com/smartcontractkit/chainlink/v2/core/utils/big_math"
+)
+
```

Comment:
maybe a better name for this file is utils.go

models.go is usually for exported interfaces or structs

Reply:
`utils.go` is better naming